components/CollectivePickerAsync.js

```diff
@@ -153,6 +153,7 @@ CollectivePickerAsync.propTypes = {
   creatable: PropTypes.bool,
   /** If true, a permanent option to invite a new user will be displayed in the select */
   invitable: PropTypes.bool,
+  onInvite: PropTypes.func,
```

Comment:
@kewitz : One another minor thing, don't think this prop is used anywhere; maybe we can remove it? 🤔 

```suggestion
```